drivers/si70xx/si70xx.c

```diff
@@ -77,9 +80,9 @@ int si70xx_init(si70xx_t *dev, i2c_t i2c_dev, uint8_t address)
     }
 
     /* initialize the peripheral */
-    i2c_acquire(dev->i2c_dev);
-    i2c_write_byte(dev->i2c_dev, dev->address, SI70XX_RESET);
-    i2c_release(dev->i2c_dev);
+    i2c_acquire(SI70XX_I2C);
+    i2c_write_byte(SI70XX_I2C, SI70XX_ADDR, SI70XX_RESET);
+    i2c_release(SI70XX_I2C);
```

Comment:
further: IMHO its fine to hold the device throughout initialization, and release it only on failure or success at the end.